dbconfig.js

```diff
@@ -0,0 +1,13 @@
+const mssql = require("mssql/msnodesqlv8");
+
+const config = {
```

Comment:
all values should be moved to `.env` file
This will allow to use different values on different servers and machines 
https://nodejs.dev/learn/how-to-read-environment-variables-from-nodejs

Reply:
Fixed. Mostly.
Whenever I move the server value into env and use the variable name instead, the app crashes.
The server value goes like this: "(LocalDB)\\MSSQLLocalDB".

Reply:
`DB_SERVER='(LocalDB)\MSSQLLocalDB'` or `DB_SERVER="(LocalDB)\\MSSQLLocalDB"`
https://www.w3schools.com/js/js_strings.asp section `Escape Character`

Reply:
Fixed.